app/src/main/res/values/strings.xml

```diff
@@ -150,22 +150,22 @@
     <string name="edit_owner_only_anon">Only the owner of this collection is allowed to edit it.</string>
     <string name="members_old_journals_not_allowed">Sharing of old-style journals is not allowed. In order to share this journal, create a new one, and copy its contents over using the \"import\" dialog. If you are experiencing any issues, please contact support.</string>
     <string name="use_native_apps_title">Did you know?</string>
-    <string name="use_native_apps_body">EteSync seamlessly integrates with Android, so to use it, just use your existing address book and calendar apps!\n\nFor more information, please check out the user guide.</string>
+    <string name="use_native_apps_body">EteSync integrates with your existing address book and calendar apps.\n\nPlease check out the user guide for more info.</string>
```

Comment:
This one is also worse than the one before. The reason why this string is there is because users were confused about how to use EteSync, so it's crucial to immediately tell them that they can just use their regular apps.

Reply:
Both strings do that, the new one without the words "seamlessly" and "Android" and the exclamation mark. Why wouldn't it?
What sort of address book and calendar apps could it be that aren't in Android? I avoided two commas, complexity, and made the string shorter, and it isn't better?

Reply:
I'm not against changing this string, I just think the explicit "to use it, use your existing..." adds value (even though it's implied in "integrates"). If you can find a way to incorporate that into the first sentence that would be great.

Reply:
"to use it, use your existing" is a horrendous construction. Double use, and double referencing "your" and "existing". One of those makes it stand out, doing it twice takes the punch out of it.